lib/core/engine.js

```diff
@@ -124,6 +124,15 @@ class Engine {
       options.postScript.unshift(...videoPostScripts);
     }
 
+    if(options.experimental.dumpChromeNetlog) {
```

Comment:
Can you change that to use lodash.get or add an extra if, yargs doesn't populate experimental so it will try .dumpChromeNetlog on an undefined object if you don't send any experimental arguments. same in chrome.js.